src/main/java/seedu/address/logic/commands/AppendCommand.java

```diff
@@ -0,0 +1,115 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FEED_TIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MEDICAL_CONDITION;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_ANIMALS;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.animal.Animal;
+import seedu.address.model.animal.Id;
+import seedu.address.model.animal.Name;
+import seedu.address.model.animal.Species;
+import seedu.address.model.feedtime.FeedTime;
+import seedu.address.model.medicalcondition.MedicalCondition;
+
+/**
+ * Edits the details of an existing animal in the zookeep book.
```

Comment:
Maybe can add on that it appends the details? Same for the `ReplaceCommand` javadocs